_data/sites.json

```diff
@@ -8382,15 +8382,6 @@
         ]
     },
 
-    {
-        "name": "Instacart",
-        "url": "https://www.instacart.com/user_privacy/form/",
-        "difficulty": "easy",
-        "domains": [
-            "instacart.com"
-        ]
-    },
```

Comment:
Is the removal of instacart intentional?

Reply:
I deleted it because i saw two versions of it in visual studio, maybe it was a bug or my folder wasnt up to date